subcontracted_service/models/procurement.py

```diff
@@ -10,46 +10,24 @@ class ProcurementOrder(models.Model):
     _inherit = "procurement.order"
 
     @api.model
```

Comment:
not sure we have to keep this decorator.
maybe `@api.multi` with `self.ensure_one()` is better. What do you think ?